src/Core/Entity/TargetedSurveyState.php

```diff
@@ -23,6 +23,12 @@ class TargetedSurveyState extends StaticEntity
 	protected $survey_status;
 	protected $message_id;
 
+	const STATUS_INVALID = 'INVALID';
+	const PENDING_RESPONSE = 'PENDING RESPONSE';
+	const RECEIVED_RESPONSE = 'RECEIVED RESPONSE';
+	const SURVEY_FINISHED = 'SURVEY FINISHED';
+	const INVALID_CONTACT_MOVED = 'ACTIVE CONTACT IN SURVEY ###';
```

Comment:
@willdoran instead of INVALID , @kinstelli suggested we set something like this with the survey number. I was for it initially but not sure now. I guess it could be useful?what do you think?

Reply:
If it's not hard to implement then it might be neat extra info to have that we could convert later but if it is hard then don't. Because we can add it as an upgrade later. If they move a contact that's their choice.

Reply:
ok awesome. I'll add it should be a minute